front/src/utils/date.ts

```diff
@@ -242,3 +242,22 @@ export const formatDateString = (date?: Date | null) => {
   if (!date) return '';
   return dayjs(date).format('DD/MM/YY');
 };
+
+const parseDate = (dateString: string): Date => {
+  const [datePart, timePart] = dateString.split(' ');
+  const [day, month, year] = datePart.split('/').map((part) => part.padStart(2, '0'));
+  const isoDateString = `${year}-${month}-${day}T${timePart || '00:00:00'}`;
+  return new Date(isoDateString);
+};
+
+export const formatDepartureDate = (date: string): string => {
```

Comment:
This sounds very specific to the `Timetable` component? If so, I'd prefer to keep it in that component.

Reply:
done

Reply:
Well, yeah, I'm not in love with the status quo I must admit. To me, there's no point in moving functions to a separate file if they're only used from a single component, especially for one-liners. It just creates more indirection IMHO.

Reply:
To expand on this a bit: `dayjs` is a time library, should we really create our own wrapper library on top of it, if it's just going to define one function per date format?